src/discussions/common/AuthorLabel.jsx

```diff
@@ -35,6 +35,11 @@ const AuthorLabel = ({
     authorLabelMessage = intl.formatMessage(messages.authorLabelStaff);
   }
 
+  if (authorLabel === 'Moderator') {
+    icon = School;
+    authorLabelMessage = intl.formatMessage(messages.authorLabelModerator);
+  }
+
```

Comment:
avoid multiple if statements here. optimize the code like this 

const labelMappings = {
  Staff: {
    icon: Institution,
    authorLabelMessage: intl.formatMessage(messages.authorLabelStaff),
  },
  Moderator: {
    icon: School,
    authorLabelMessage: intl.formatMessage(messages.authorLabelModerator),
  },
  'Community TA': {
    icon: School,
    authorLabelMessage: intl.formatMessage(messages.authorLabelTA),
  },
};

const labelInfo = labelMappings[authorLabel] || {};
const { icon, authorLabelMessage } = labelInfo;